spec/decorators/course_decorator_spec.rb

```diff
@@ -4,6 +4,10 @@
   let(:current_recruitment_cycle) { build :recruitment_cycle }
   let(:next_recruitment_cycle) { build :recruitment_cycle, :next_cycle }
   let(:provider) { build(:provider, accredited_body?: false) }
+  let(:english) { build(:subject, subject_name: "English") }
+  let(:mathematics) { build(:subject, subject_name: "Mathematics") }
```

Comment:
Would it makes sense for these to be traits on the `subject` factory? Seems to me that it'd reduce verbosity and chance or error.